Bugfix and old links removal

Trainax opened the pull request:
Rimosso il link al sito dell'INRiM dal file dato che il segnale orario non viene più generato da loro.
Aggiunto il link alla pagina di Wikipedia nel README.
I nomi dei giorni della settimana non venivano visualizzati correttamente:

![Giorni SRC old](https://user-images.githubusercontent.com/30187360/66229531-52f07b00-e6e2-11e9-88eb-4af991c5037f.PNG)

I nomi dei giorni sono quindi stati modificati per poter essere visualizzati correttamente.
Ora vengono visualizzati cosi:

![Giorni SRC new](https://user-images.githubusercontent.com/30187360/66229735-cb573c00-e6e2-11e9-8b90-d043080367a2.png)

Trainax committed:
Bugfix and old links removal

Rimosso il link al sito dell'INRiM dal file dato che il segnale orario non viene più generato da loro.
Aggiunto il link alla pagina di Wikipedia nel README.
I nomi della settimana sono stati modificati per poter essere visualizzati correttamente

Trainax committed:
README fix

Sistemato un link nel README.
Migliore messaggio per la scelta del giorno della settimana

zarelit commented:
Grazie della PR @Trainax !
Ho aperto una issue separata per risolvere la visualizzazione dei giorni della settimana perché vorrei capire quale è il problema vero per provare a risolverlo senza troncare i nomi dei giorni oppure se non si riesce a capire troncarli tutti a tre lettere (inclusi sabato e domenica), quindi farei una PR separata per quello.

Il link dell'INRIM è morto e quello di wikipedia va più che bene. Ho trovato [un altro URL dell'INRIM](http://toi.inrim.it/ist/it/iensrc.html) che contiene la spiegazione del protocollo e timing diagrams, però a questo punto non mi fido della stabilità degli URL, forse si potrebbe mettere un link al web archive? 

EDIT: messo il link corretto

Che ne pensi?

zarelit commented on INRIM_SRC.ny:
```diff
@@ -12,16 +12,16 @@
 ;control mese "Mese" int "" 4 1 12
 ;control giorno "Giorno" int "" 22 1 31
-;control settimana "Nome giorno" choice "lunedì,martedì,mercoledì,giovedì,venerdì,sabato,domenica" "venerdì"
+;control settimana "Nome del giorno" choice "Lun.,Mart.,Merc.,Giov.,Ven.,Sabato,Domenica" "Sabato"
```
purtroppo "Sabato" crea un SRC inconsistente perché la data specificata negli altri default indica un venerdì. Quindi o si rimette `Ven.` o si mette il giorno del mese a 23 invece di 22 :smiley:

Trainax replied:
Hai ragione, ho messo "Sabato" per evitare problemi dovuti alle strane lettere accentate senza pensare all'effettiva corrispondenza tra la data e l'effettivo giorno della settimana. Io a questo punto rimetterei momentaneamente "Ven." finché non riusciamo a risolvere il problema dei caratteri accentati non visualizzati correttamente. Questo problema secondondo me è dovuto al fatto che Audacity non usa una codifica di caratteri contenente le lettere accentate e quindi quello è il risultato...

Edit: vorrei aggiungere che lo screenshot è stato fatto usando l'ultima versione di Audacity

Trainax commented:
Per quanto riguarda il link io non metterei quello diretto al sito dell'INRiM visto che potrebbero togliere la pagina senza preavviso. Il [link alla pagina salvata su Internet Archive](https://web.archive.org/web/20161223104950/http://www.inrim.it:80/res/tf/src_i.shtml) potrebbe essere una buona idea, peccato per un link non funzionante ad un'immagine che presumo in origine fosse [questa](https://3.bp.blogspot.com/-Zsn04simRaM/WGTO6DqHHoI/AAAAAAAAGzQ/OQ-WQIuxFtk8hpEqhPPwgnW_5xJNeDlcgCLcB/s1600/src.jpg).

Quindi per ora preferisci fare un commit solo per correggere i link nel README e poi fare un altro commit quando si riuscirà a capire come sistemare le lettere accentate?

Edit: ho rimesso "Ven." così il segnale generato ora è corretto. Ho abbreviato i nomi dei giorni a 3 lettere come avevi suggerito nel frattempo che riusciamo a capire come far visualizzare correttamente i caratteri accentati. Ho aggiunto nel README il link alla pagina archiviata dal sito dell'INRiM

Trainax committed:
Bugfix e aggiunta informazioni

Corretto un problema che provocava la generazione di un segnale SRC inconsistente.
Tutti i nomi dei giorni della settimana sono stati abbreviati a 3 lettere.
Aggiunta del link alla pagina archiviata dal sito dell'INRiM nel README
